docs/src/modules/concepts/pages/grpc-vs-http-endpoints.adoc

```diff
@@ -1,6 +1,8 @@
 = Endpoints
 
-Endpoints components are how you expose your services to the outside world, be it another Akka service, external dependencies or client-facing applications. Two different types of endpoints are available: HTTP and gRPC Endpoints.
+Endpoints components are how you expose your services to the outside world, be it another Akka service, external dependencies or client-facing applications. Three different types of endpoints are available: HTTP, gRPC and MCP Endpoints.
+
+MCP endpoints are only for providing functionality to a remote agent and not for defining regular APIs, for more details see xref:java:mcp-endpoints.adoc[]
```

Comment:
Shall we also spell out the MCP abbreviation, since this might be the first time reading that?